english/call-a-doctor/hungry/hungry.md

```diff
@@ -3,15 +3,17 @@
 "But I'm so hungry that the walls are turning to marshmallows!" you say to the doctor.
 
 "Marshmallows, huh?" He scribbles something on a page in a folder thick with
-documents. "Yesterday, you said you were drowning in gravy. The day before that, 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.